### PR TITLE
fix: added missing environmentID flag

### LINKED ISSUE #197: unknown flag: --environmentID  for litmusctl connect chaos-infra
parameter environmentID is missing here :
https://github.com/litmuschaos/litmusctl/blob/master/pkg/cmd/connect/infra.go#L268

### PR DESCRIPTION
**Pull request by kartikaysaxena**
Should fix #197

### CODE REVIEW

**Comment by kartikaysaxena** [comment]
The flag had been implemented earlier [here](https://github.com/litmuschaos/litmusctl/blob/master/pkg/cmd/connect/infra.go#L108), just the flag parser was missing, I have added that.

**Review by neelanjan00 — APPROVED** [review]

**Comment by SarthakJain26** [comment]
@kartikaysaxena can you please update the flag to environment-id, since this is the casing that we are following. Also please update the example command in connect/connect.go by adding the environment-id flag.

**Comment by kartikaysaxena** [comment]
> @kartikaysaxena can you please update the flag to environment-id, since this is the casing that we are following. Also please update the example command in connect/connect.go by adding the environment-id flag.

Sure will do it.

File: pkg/cmd/connect/connect.go [hunk 1]
```diff
@@ -25,10 +25,10 @@ var ConnectCmd = &cobra.Command{
 	Short: `Connect resources for LitmusChaos Execution plane.
 		Examples:
 		#connect a Chaos Infrastructure
-		litmusctl connect chaos-infra --name="new-chaos-infra" --non-interactive
+		litmusctl connect chaos-infra --name="new-chaos-infra" --non-interactive --environment-id="my-environment-id"
```
Comment (SarthakJain26) [inline pkg/cmd/connect/connect.go:28]: No need to add incase of interactive mode, user will see a prompt to enter the environment-id incase of interactive mode

**Review by SarthakJain26 — CHANGES_REQUESTED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Pull request merged**